pkg/sink/sink.go

```diff
@@ -554,7 +558,28 @@ func (r Sink) ExecuteInterceptors(trInt []*triggersv1.TriggerInterceptor, in *ht
 		}
 		// Clear interceptorParams for the next interceptor in chain
 		request.InterceptorParams = map[string]interface{}{}
+
+		// parse form-data payload for slack only
+		if v := in.Header.Get("X-Slack-Signature"); v != "" {
```

Comment:
can we do this generally for all urlencoded form bodies? i.e. if the content type is urlencoded, we can parseQuery and use the jsonified string as the body?

Reply:
yes